src/PrestaShopBundle/Resources/config/services/core/form/form_handler.yml

```diff
@@ -213,3 +213,9 @@ services:
     factory: 'prestashop.core.form.identifiable_object.handler.form_handler_factory:create'
     arguments:
       - '@prestashop.core.form.identifiable_object.data_handler.zone_form_data_handler'
+
+  prestashop.core.form.identifiable_object.handler.search_engine_form_handler:
+    class: 'PrestaShop\PrestaShop\Core\Form\IdentifiableObject\Handler\FormHandler'
```

Comment:
indentation

Reply:
please check all your yml files for indention (I know some of them are not following all the rules, but please make at least same as older services to keep it clean, I hope in future we will have some linter to fix all of yml files indent automatically)